### PR TITLE
adjusted guide for kubernetes 1.7.6

### PR DESCRIPTION
**Pull request by auhlig**
While trying this with kubernetes 1.7.6 ..
/cc @SchwarzM

### CODE REVIEW

**Comment by urfuwo** [comment]
we should change the guide to 1.7.6 or the latest we test with in general please

**Comment by auhlig** [comment]
TODO: Adjust kube-dns part.

**Comment by auhlig** [comment]
Cluster is up and running. Thanks @jonathanschwarze, @SchwarzM.
Good to merge @BugRoger?

File: docs/02-certificate-authority.md [hunk 1]
```diff
@@ -54,6 +54,13 @@ chmod +x cfssljson_linux-amd64
 sudo mv cfssljson_linux-amd64 /usr/local/bin/cfssljson
 ```
 
+**Note:** On CoreOS the above directory is read only, therefore you might want to use a different path.
+```
+sudo mkdir -p /opt/bin/
+sudo mv cfssljson_linux-amd64 cfssl_darwin-amd64 /opt/bin/
```
Comment (BugRoger) [inline docs/02-certificate-authority.md]: Something odd going on. I think, I did this on my Mac when creating the guide. Copying the linux binary as darwin? Needs to be cleaned up to be Darwin or Linux...

Comment (auhlig) [reply docs/02-certificate-authority.md]: Ah. Yes. Will fix.

File: docs/02-certificate-authority.md [hunk 2]
```diff
@@ -29,15 +29,15 @@ This lab requires the `cfssl` and `cfssljson` binaries. Download them from the [
 ### OS X
 
 ```
-wget https://pkg.cfssl.org/R1.2/cfssl_darwin-amd64
-chmod +x cfssl_darwin-amd64
-sudo mv cfssl_darwin-amd64 /usr/local/bin/cfssl
+wget https://pkg.cfssl.org/R1.2/cfssl_linux-amd64
```
Comment (SchwarzM) [inline docs/02-certificate-authority.md:32]: this does look like a linux binary and not darwin ...

File: docs/03-auth-configs.md [hunk 3]
```diff
@@ -11,7 +11,7 @@ The kubectl client will be used to generate kubeconfig files which will be consu
 ### OS X
 
 ```
-wget https://storage.googleapis.com/kubernetes-release/release/v1.6.0/bin/darwin/amd64/kubectl
+wget https://storage.googleapis.com/kubernetes-release/release/v1.6.0/bin/linux/amd64/kubectl
```
Comment (SchwarzM) [inline docs/03-auth-configs.md:14]: s/linux/darwin/g

File: docs/04-kubectl.md [hunk 4]
```diff
@@ -7,26 +7,33 @@ Run the following commands from the machine which will be your Kubernetes Client
 ### OS X
 
 ```
-wget https://storage.googleapis.com/kubernetes-release/release/v1.6.3/bin/darwin/amd64/kubectl
+wget https://storage.googleapis.com/kubernetes-release/release/v1.7.6/bin/linux/amd64/kubectl
```
Comment (SchwarzM) [inline docs/04-kubectl.md:10]: s/linux/darwin/g

**Pull request merged**